src/SprykerEco/Yves/Computop/ComputopConfigInterface.php

```diff
@@ -1,5 +1,4 @@
 <?php
-//phpcs:ignoreFile
```

Comment:
ECO is striking back :) Why do we have an interface for the config is the question that I guess no one can answer right now :)